### PR TITLE
Aman/changes

### PR DESCRIPTION
**Pull request by ataddesse**
chrome-capture (3)

## Description

This PR:
 - Modifies guide page UI
 - Fixes map get user location bug
 - Removes unused imports  

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (change that introduces a bug or causes app to crash, but that we plan on fixing later)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Provide any test suites/filenames that you had to create/change.

- [ ] Test A
- [ ] Test B

## Checklist

- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

NB: I have merged revert Roddy/fix i18n so only consider changes on the following files
        modified:   src/app/components/crm/analytics/map/map.component.ts
        modified:   src/app/components/crm/campaigns/campaigns.component.ts
        modified:   src/app/components/crm/guide/guide.component.html
        modified:   src/app/components/crm/guide/guide.component.ts
        modified:   src/app/components/crm/guide/guide.module.ts
        modified:   src/app/services/login.service.ts

### CODE REVIEW

File: frontend/src/app/components/crm/analytics/map/map.component.ts [hunk 1]
```diff
@@ -54,19 +53,28 @@ export class MapComponent implements AfterViewInit {
...
     this.heatmap = new google.maps.visualization.HeatmapLayer({
       data: this.loadAllHeatLocations(),
       map: this.map
     });
 
+    // Get user's location if possible
+    const userLocation = new Promise<any>((resolve, reject) => {
+
+      navigator.geolocation.getCurrentPosition(resp => {
+
```
Comment (anttsou-google) [inline frontend/src/app/components/crm/analytics/map/map.component.ts:67]: remove extra newline

File: frontend/src/app/components/crm/analytics/map/map.component.ts [hunk 2]
```diff
@@ -54,19 +53,28 @@ export class MapComponent implements AfterViewInit {
...
 
+    // Heatmap loading starts here
     this.heatmap = new google.maps.visualization.HeatmapLayer({
       data: this.loadAllHeatLocations(),
       map: this.map
     });
 
+    // Get user's location if possible
+    const userLocation = new Promise<any>((resolve, reject) => {
+
```
Comment (anttsou-google) [inline frontend/src/app/components/crm/analytics/map/map.component.ts:65]: remove extra newline

File: frontend/src/app/components/crm/analytics/map/map.component.ts (hunk 1 above)
Comment (ataddesse) [reply frontend/src/app/components/crm/analytics/map/map.component.ts]: alright, fixed.

File: frontend/src/app/components/crm/analytics/map/map.component.ts (hunk 2 above)
Comment (ataddesse) [reply frontend/src/app/components/crm/analytics/map/map.component.ts]: fixed

**Pull request merged**